Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -524,6 +524,26 @@ class GetVariable final : public Instruction {
     Optional<EnvironmentCoordinate> mutable m_cached_environment_coordinate;
 };
 
+class GetGlobal final : public Instruction {
+public:
+    explicit GetGlobal(IdentifierTableIndex identifier, u32 cache_index)
+        : Instruction(Type::GetGlobal)
+        , m_identifier(identifier)
+        , m_cache_index(cache_index)
+    {
+    }
+
+    ThrowCompletionOr<void> execute_impl(Bytecode::Interpreter&) const;
+    DeprecatedString to_deprecated_string_impl(Bytecode::Executable const&) const;
+    void replace_references_impl(BasicBlock const&, BasicBlock const&) { }
+    void replace_references_impl(Register, Register) { }
+
+private:
+    IdentifierTableIndex m_identifier;
+    u32 m_cache_index { 0 };
+    mutable u64 m_unique_global_declarative_environment_serial_number { 0 };
```

Comment:
Let's store this with the executable instead. The instruction stream is pleasantly immutable today, and it'd be nice to keep it that way :)